packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -134,10 +139,11 @@ contract L2StandardBridge is StandardBridge {
     ) internal {
         address l1Token = OptimismMintableERC20(_l2Token).l1Token();
         if (_l2Token == Lib_PredeployAddresses.OVM_ETH) {
+            require(msg.value == _amount, "ETH withdrawals must include sufficient ETH value.");
```

Comment:
It would be nice to cover this `require` statement in this PR, something that would fail before but pass now

Reply:
test added: 
https://github.com/ethereum-optimism/optimism/blob/1f8518b0e5d982453e8cf623ae293662d0e01467/packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol#L49